### PR TITLE
feeRate Enhancement plus minor fixes

### PR DESCRIPTION
**Pull request by AliMeer**
- update satark contract address 
- feeRate from config file
- ledger registration bug fix
- getVaultId added for ledger deposit

### CODE REVIEW

File: src/api/deposit.js [hunk 1]
```diff
@@ -12,10 +12,8 @@ module.exports = async (dvf, token, amount, starkPrivateKey) => {
   const tempVaultId = dvf.config.DVF.tempStarkVaultId
   const nonce = dvf.util.generateRandomNonce()
   const starkTokenId = currency.starkTokenId
-  let starkVaultId = currency.starkVaultId
-  if (!starkVaultId) {
-    starkVaultId = dvf.config.spareStarkVaultId
-  }
+  const starkVaultId = await dvf.getVaultId(token)
+
```
Comment (plutoegg) [inline src/api/deposit.js:16]: Why is git showing this change? This is already on dev branch

File: src/api/register.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ module.exports = async (dvf, starkPublicKey) => {
 
   const { deFiSignature } = await post(url, { json: data })
 
-  const onchainRegister = true //await dvf.stark.register(dvf, starkKey, deFiSignature)
+  const onchainRegister = await dvf.stark.register(dvf, starkKey, deFiSignature)
```
Comment (plutoegg) [inline src/api/register.js:17]: So is this

File: src/api/deposit.js (hunk 1 above)
Comment (AliMeer) [reply src/api/deposit.js]: I have rebased now its not showing this change.

File: src/api/register.js (hunk 2 above)
Comment (AliMeer) [reply src/api/register.js]: Same for this one. Rebased so this is not coming up as a change now.

**Pull request merged**